docs/content/docs/try-flink-kubernetes-operator/quick-start.md

```diff
@@ -63,22 +63,15 @@ minikube start --kubernetes-version=v1.21.5
 
 We also recommend [k9s](https://k9scli.io/) as GUI for kubernetes, but it is optional for this quickstart guide.
 
-## Building the operator image from source
+## Get the source code
```

Comment:
Should we use `## Getting the source code` to follow our naming pattern of each section?